config/checkstyle/checkstyle.xml

```diff
@@ -162,7 +162,6 @@
 
     <module name="LeftCurly">
```

Comment:
Can make it self close since its empty: `<module name="LeftCurly"/>` and move comments on top.